classes/controller/FrontController.php

```diff
@@ -1940,6 +1940,11 @@ public function getAssetUriFromLegacyDeprecatedMethod($legacy_uri)
         }
     }
 
+    public function displayAjax()
+    {
+        $this->assignGeneralPurposeVariables();
```

Comment:
note: this don't fix this bug specifically but I have the feeling that these variables are not available everywhere in ajax. I've played with front office a little bit and I can't find an ajax call that is broken after this addition :)